app/tests/controllers/course/course_controllers_utils.py

```diff
@@ -0,0 +1,26 @@
+# @author Simone Nicol <en0mia.dev@gmail.com>
+# @created 19/07/23
+import uuid
+
+from app.stuquiz.entities.category import Category
+from app.stuquiz.entities.course import Course
+
+
+class CourseControllersUtils(object):
+    @staticmethod
+    def generate_expected_body(courses: list[Course]) -> list[dict]:
+        return [{
+            'id': course.id,
+            'university_id': course.university_id,
+            'name': course.name,
+            'description': course.description,
+            'professor_id': course.professor_id,
+            'categories': [category.dump() for category in course.categories],
+            'code': course.code
+        } for course in courses]
+
+    @staticmethod
+    def generate_courses(number: int) -> list[Course]:
+        return [Course(str(uuid.uuid4()), str(uuid.uuid4()), f"Test course {i}",
+                       f"Test course {i} description", str(uuid.uuid4()), f"TEST{i}",
```

Comment:
Here you've cast a uuid in string, but we can define professor_id as uuid at thhe beginning